setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 
-version = '0.4.1b13'
+version = '0.4.1b14'
```

Comment:
Bump version. The IdP now in eduid-IdP master can work with this version; so we can deploy this PR first, and then eduid-IdP's.